docs/scos/dev/back-end-development/extend-spryker/extend-spryker.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Extend Spryker
+description: This section lists documents inside the Extend Spryker section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Extend Spryker* section includes the following documents:
```

Comment:
Can you add a few words on what this section is about?

Reply:
```suggestion
The *Extend Spryker* section section provides guidance on extending Spryker to enhance its functionality. In this section, you can find documents about creating modules for encapsulating new concepts and learn about extending a core module used by another core module. You explore different approaches such as replacement class, inheritance object, and composition to extend the core. Also, this section shows how to extend the Spryker Core functionality, use the Spryker Engine and learn how to install the module structure and configuration.

In this section, you can find the following documents and subsections:
```
```